### PR TITLE
net-irc/hexchat: add 2.16.2

### PR DESCRIPTION
**Pull request by NexAdn**

### CODE REVIEW

File: net-irc/hexchat/hexchat-2.16.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+				dev-lang/mono[minimal]
+				dev-dotnet/libgdiplus
+			)
+			dev-lang/mono[-minimal]
+		)
+	)"
+
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-util/glib-utils
```
[the remark this answers is not in the document]
Comment (NexAdn) [reply net-irc/hexchat/hexchat-2.16.2.ebuild]: done

**Comment by NexAdn** [comment]
Requested changes are applied now

**Review by ceamac — APPROVED** [review]
What's left is mostly a matter of style, you can do the changes in a follow-up PR if you want.  I will merge this one.

Thank you!

File: net-irc/hexchat/hexchat-2.16.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+pkg_preinst() {
+	if use gtk ; then
+		xdg_pkg_preinst
+	fi
+}
+
+pkg_postinst() {
+	if use gtk ; then
+		xdg_pkg_postinst
```
Comment (ceamac) [inline net-irc/hexchat/hexchat-2.16.2.ebuild:115]: (I think I didn't click something so my comment here was lost.  I'll do it again.)

`xdg_pkg_postinst` does nothing if ! use gtk.  I would move it out of the if block and also remove `pkg_preinst` and `pkg_postrm`, the `xdg` eclass can handle it.

**Comment by thesamesam** [comment]
I suppose live could even go now.

**Pull request closed without merging**